Keep project name display aligned

guessi opened the pull request:
## What this PR does / why we need it:

To keep project name display aligned across all documentations.

** To keep minimal risk of change, the PR will focus on `*.md` files only.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] CVE Report (Scanner found CVE and adding report)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation only

## Which issue/s this PR fixes

Please refer to #9918 for more details.

## How Has This Been Tested?

n/a

## Checklist:
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/kubernetes/ingress-nginx/blob/main/CONTRIBUTING.md) guide
- [ ] I have added unit and/or e2e tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Added Release Notes.

## Does my pull request need a release note?

No

guessi commented on mkdocs.yml:
```diff
@@ -1,4 +1,4 @@
-site_name: NGINX Ingress Controller
+site_name: NGINX-Ingress Controller
```
This is the only non-md update in this PR.

I believe it should be the title of the [Project Homepage](https://kubernetes.github.io/ingress-nginx/), should be aligned as well.

longwuyuan commented:
@guessi thanks for the contribution

- I think Project Title change also is ok
- Please ignore changelog in this PR
- The README inside the charts directory is autogenerated using helm-docs https://github.com/kubernetes/ingress-nginx/blob/main/MANUAL_RELEASE.md#d-edit-the-valuesyaml-and-run-helm-docs

guessi commented on mkdocs.yml:
```diff
@@ -1,4 +1,4 @@
-site_name: NGINX Ingress Controller
+site_name: Ingress-Nginx Controller
```
This is the only non-md update in this PR.

I believe it should be the title of the [Project Homepage](https://kubernetes.github.io/ingress-nginx/), should be aligned as well.

guessi commented:
@longwuyuan I've reverted the changes to `Changelog.md` as requested. Please review the changes again, thanks!

longwuyuan commented:
@guessi please ceck my comment earlier about the README inside the carts directory

guessi commented:
@longwuyuan I've updated the `charts/*` but it seems like failed with unrelated test cases. Please review while you are free, thanks!

longwuyuan commented:
@guessi can you leave the /charts directory out of the PR for now. We can add it after the tests pass. Te flaking tests are a known issue so please remember to keep rebasing if main head gets ahead.

guessi commented:
Sure, no problem, will update in minutes.

longwuyuan commented:
later when you do want to change the README under /charts, please read the link I posted earlier about helm-docs tool. It means you have to edit the values yaml and use the helm docs tools to generate the README file. That file is not to be edited manually

guessi commented:
Just updated, I removed the changes to `charts/*`.

Actually, I do follow the [instructions](https://github.com/kubernetes/ingress-nginx/blob/main/MANUAL_RELEASE.md#d-edit-the-valuesyaml-and-run-helm-docs) to update `values.yaml` only. Here's what I did,

```bash
$ vim values.yaml
$ helm-docs ...
$ git diff --exit-code
$ git add charts/*
$ git comit ...
```

but I have no idea why it still failed.

guessi commented:
For what I see, most recent PRs are all now failing at the same check items, I believe there might have something went wrong before the PR created. Anyway... I've removed all the changes to `charts/*`, let's see what would happen.

longwuyuan commented:
Thanks @guessi 
/ok-to-test

longwuyuan commented:
/triage-accepted
/kind documentation
/lgtm

guessi commented:
Thanks for merging this. One quick question, should I create a new PR for `charts/*` or maybe you guys would do that?

Let me know what I could helps.

longwuyuan commented:
@guessi try new PR for /charts/* .  You would need to edit values.yaml and then use helm-docs exactly as instructed .
